### DISCUSSION TITLE
Package always empty

### DISCUSSION THREAD

**Discussion by vanillaHill** [semantic-release/semantic-release#2412] [Q&A] [answered]
Hello, 
Any idea what I'm doing wrong with semantic-release ? It doesn't take into account my lib repo.
My repo (lib is created after the build): 
<img width="295" alt="Capture d’écran 2022-04-07 à 13 56 57" src="https://user-images.githubusercontent.com/34197445/162194058-b245011c-94c0-4898-8920-ed85c13093ab.png">
My package deployed (empty):
<img width="291" alt="Capture d’écran 2022-04-07 à 13 58 03" src="https://user-images.githubusercontent.com/34197445/162193839-1c9f670e-b67a-45dd-8663-10d5ffab7cea.png">

Thanks.

**Comment by travi** [comment, marked as answer]
semantic-release does not define which files get included into your package. instead you need to define that information by configuring npm. i see that you have a `.gitignore` file. does that exclude your `lib/` directory, by chance? in the absence of a `.npmignore` file or a `files` block in your `package.json` (our recommendation), npm will fall back to the information provided in the `.gitignore`

**Reply by vanillaHill** [reply]
I confirm, .gitignore ignores the repo (that's the problem).
Add .npmignore fixes the issue.
Thank you sir.
